Jint/Native/TypedArray/TypedArrayInstance.cs

```diff
@@ -330,5 +331,23 @@ private static bool IsIntegralNumber(double value)
                    && !double.IsInfinity(value)
                    && System.Math.Floor(System.Math.Abs(value)) == System.Math.Abs(value);
         }
+
+        internal T[] ToNativeArray<T>()
+        {
+            var conversionType = typeof(T);
+            var elementSize = _arrayElementType.GetElementSize();
```

Comment:
I believe this could be a static field?

Reply:
super nit right?

Reply:
Size is calculated via switch as there's only one TypeArrayInstance type and no specific for each one, it should be quite cheap...

Reply:
I agree. It just reminded me this trick that things on the generic type can be done statically without interfering between types.